src/cli/argument-parser.js

```diff
@@ -111,6 +111,7 @@ export default class CLIArgumentParser {
             .option('--proxy-bypass <rules>', 'specify a comma-separated list of rules that define URLs accessed bypassing the proxy server')
             .option('--disable-page-reloads', 'disable page reloads between tests')
             .option('--qr-code', 'outputs QR-code that repeats URLs used to connect the remote browsers')
+            .option('--ignore-uncaught-errors', 'ignore uncaught errors')
```

Comment:
I guees it will be development option that turns on capturing unhandled rejection and uncaught errors, run TestCafe with unminified scripts and show full callstacks (in the future).
@kirovboris @AndreyBelym 

So, the suitable name for this - `--doctor-mode` or `--development` or something else.

Reply:
I think we should provide a dedicated option to disable the uncaught exceptions/unhandled Promise rejections handler for server-side test code, like we do with the `--allow-js-errors` for client-side test code. E.g. `mocha` provides the [allowUncaught](https://github.com/mochajs/mocha/blob/668b541090c84b66a701823b30f6880d3fc0f2b9/test/unit/mocha.spec.js#L158) option for this purpose.

We can of course disable server-side error handler together with the client-side one by using the `--allow-js-error` switch, but I would prefer a dedicated, because an uncaught exception on the client-side can be raised by a tested page, but an uncaught exception on the server-side clearly indicates that there is a problem/mistake in the tests' code.

Reply:
Need to discussing with @kirovboris

Reply:
We discussed it in private and we should do some checks with real existing tests, because it's we need to be sure the throwing unhandledRejaction will works in all base scenarios and won't create a breaking change. If we have any  troubles, of cource we should keep it behind some `debug` option.